### PR TITLE
User admin e2e tests

### PR DESCRIPTION
**Pull request by evansiroky**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

In anticipation of potentially moving some user data from Auth0 to mongo, I have made some e2e tests for user admin items.

### CODE REVIEW

File: __tests__/end-to-end.js [hunk 1]
```diff
@@ -589,7 +607,7 @@ describe('end-to-end', () => {
       } else log.warn('OpenTripPlanner not accepting requests. Start it up for deployment tests!!')
     }
     log.info('Launching chromium for testing...')
-    browser = await puppeteer.launch(puppeteerOptions)
+    browser = await launch(puppeteerOptions)
```
Comment (landonreed) [inline __tests__/end-to-end.js]: I'm getting an error here when I run this locally:
```
 ● end-to-end › admin › should allow admin user to create another user

    TypeError: Cannot read property '_launcher' of undefined

      608 |     }
      609 |     log.info('Launching chromium for testing...')
    > 610 |     browser = await launch(puppeteerOptions)
          |                                     ^
      611 |     page = await browser.newPage()
```

Comment (evansiroky) [reply __tests__/end-to-end.js]: Try running `rm -rf node_modules && yarn && yarn run test-end-to-end`.

File: lib/admin/components/UserSettings.js [hunk 2]
```diff
@@ -20,7 +20,8 @@ type Props = {
   isCreating?: boolean,
   organizations: Array<Organization>,
   permissions: UserPermissions,
-  projects: Array<Project>
+  projects: Array<Project>,
+  userEmailDataSlug?: string
```
[the remark this answers is not in the document]
Comment (evansiroky) [reply lib/admin/components/UserSettings.js]: The UserSettings has some buttons that need to be clicked and in order to find the exact buttons needed, this string needs to be passed through. This is really just for testing purposes.

File: __tests__/end-to-end.js [hunk 3]
```diff
@@ -668,12 +686,83 @@ describe('end-to-end', () => {
     await wait(2000, 'for projects to load')
   }, defaultTestTimeout, 'should load the page')
 
+  describe('admin', () => {
+    const testUserEmail = `e2e-test-${fileSafeTestTime}@ibigroup.com`.toLowerCase()
```
[the remark this answers is not in the document]
Comment (evansiroky) [reply __tests__/end-to-end.js]: I don't think they'd get an email, I believe we've disabled new user creation emails in Auth0.

**Comment by evansiroky** [comment]
Let me know if nuking node_modules made it work.

File: __tests__/end-to-end.js (hunk 1 above)
Comment (landonreed) [reply __tests__/end-to-end.js]: Nope. Are you sure you don't have any local changes? It works when I import puppeteer instead of `import {launch}`

Comment (evansiroky) [reply __tests__/end-to-end.js]: Just made a little refactor. Try again.

**Pull request merged**